File: frontend/src/features/actividades/gestion_rectificatoria_solucion_reparo/models/gestion_rectificatoria_solucion_reparo.ts
export interface GestionRectificatoriaSolucionReparo {
  id_gestion_rectificatoria_solucion_reparo?: number | null;
  id_gestion_rectificatoria?: number | null;
  id_expediente?: number | null;
  id_usuario_solicitante?: number | null;
modificar_datos_memo?: boolean | null;
  descontabilizar_operacion?: boolean | null;
  subsanar?: boolean | null;
  observaciones?: string | null;
  id_solicitante:number | null;
  id_solicitud:number | null;
  solicitante?: string | null;
  observaciones_reparo?: string | null;
  fecha_ingreso?: string | null;
  
  is_active?: boolean | null;
  row_status?: boolean | null;
  created_by?: number | null;
  created_date?: string | null;
  modified_by?: number | null;
  modified_date?: string | null;
}

File: backend/Multibanca.Domain.Models/Multibanca/gestion_rectificatoria_solucion_reparo.cs
using Common.Domain.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Multibanca.Domain.Models.Multibanca
{
    public class gestion_rectificatoria_solucion_reparo : base_auditoria
    {
        public int id_gestion_rectificatoria_solucion_reparo { get; set; }
        public int id_gestion_rectificatoria { get; set; }
        public long id_expediente { get; set; }
        public int id_usuario_solicitante { get; set; }
        public bool modificar_datos_memo { get; set; }
        public bool descontabilizar_operacion { get; set; }
        public bool subsanar { get; set; }
        public string? observaciones { get; set; }
        public int id_solicitante { get; set; }
        public int id_solicitud { get; set; }
        public string? solicitante { get; set; }
        public string? observaciones_reparo { get; set; }
        public DateTime? fecha_ingreso { get; set; }
    }
}
